### ISSUE TITLE
[iOS][macOS]: OPML import appears to only import half of podcasts

### ISSUE THREAD

**Issue by gdavis** [gdavis/Mimir-Feedback#194]
User feedback: 
> When I import podcasts, either from an opml file or through the share tab, I only get about a third to import. This is from Overcast and Castro. 

Attached are the two OPML files used for this issue.
[OPML-issues.zip](https://github.com/gdavis/Mimir-Feedback/files/9329071/OPML-issues.zip)

**Comment by drallgood** [comment]
I've noticed the same thing
Can share my opml if needed

**Comment by gdavis** [comment]
I have found that there are some issues related to the placeholder record made when importing podcasts when the RSS feed that is actually loaded has a different title. I have added a fix for this where the importer will now look for a local podcast record with a matching feed URL and use the local podcast's identifier if it does not match the identifier it expected in the feed.

This resolves an issue where a second podcast record could be created, and never populated with the correct episode information so it never appeared. 

Will be part of the next release coming in the next week or so.
